XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Profiles/WindowsMixedRealityControllerDataProviderProfile.cs

```diff
@@ -59,7 +59,8 @@ public override ControllerDefinition[] GetDefaultControllerOptions()
         {
             return new[]
                {
-                 new ControllerDefinition("HoloLensController", typeof(WindowsMixedRealityMotionController)),
+                 new ControllerDefinition(typeof(HololensOneController), Handedness.Left),
```

Comment:
Can we please fix "Hololens" to "HoloLens". I don't know why but i just don't like Hololens at all :D

Reply:
The only reason we put the casing here to "Hololens" is to ensure Unity doesn't put a space in the name in the editor, which it does by default.  All other classes have the correct casing.

Reply:
Hmm ok. Not really happy that such things influnce code but not a big deal as well I guess. We should be able to specify a display name for these definitions.